api/controllers/ApplicationTypeController.js

```diff
@@ -325,7 +325,7 @@ module.exports = {
                                     }
 
                                     if (req.session.appType === 4) {
```

Comment:
I know this is not part of the change but I see magic numbers, what is '_appType === 1_' and '_appType === 4_'?

Reply:
This is a fair spot, I'm happy to change this.